From: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
Date: Mon, 27 Sep 2021 11:59:17 +0200
Subject: [PATCH] irq: Export force_irqthreads_key

Temporary add the EXPORT_SYMBOL_GPL for force_irqthreads_key until it is
settled if it is needed or not.

Signed-off-by: Sebastian Andrzej Siewior <bigeasy@linutronix.de>
---
 kernel/irq/manage.c |    1 +
 1 file changed, 1 insertion(+)

--- a/kernel/irq/manage.c
+++ b/kernel/irq/manage.c
@@ -26,6 +26,7 @@
 
 #if defined(CONFIG_IRQ_FORCED_THREADING) && !defined(CONFIG_PREEMPT_RT)
 DEFINE_STATIC_KEY_FALSE(force_irqthreads_key);
+EXPORT_SYMBOL_GPL(force_irqthreads_key);
 
 static int __init setup_forced_irqthreads(char *arg)
 {
